lms/templates/courseware/info.html

```diff
@@ -81,6 +81,23 @@ <h3 class="page-title">${_("Welcome to {org}'s {course_name}!").format(org=cours
             </div>
           % endif
 
+          % if upgrade_banner:
+              <div class="upgrade-banner layout-1q3q layout-reversed">
+                <div class="upgrade-icon">
+                  <img src="${STATIC_URL}images/edx-verified-mini-cert.png">
+                </div>
+                <div class="upgrade-msg">
+                  <h4>${_("Give yourself an additional incentive to complete")}</h4>
+                  <p>${_("Earn a verified certificate.")}
+                    <a href="https://www.edx.org/verified-certificate" target="_blank">${_("Learn More")}</a>
+                  </p>
+                </div>
+                <div class="upgrade-banner-button">
```

Comment:
If you use a span instead of a div, I think you won't need to set display to inline-block.